app/kumactl/pkg/client/api_server_client.go

```diff
@@ -1,18 +1,28 @@
 package client
 
 import (
+	"fmt"
 	"net/http"
 	"net/url"
+	"runtime"
 	"time"
 
 	"github.com/pkg/errors"
 
 	config_proto "github.com/kumahq/kuma/pkg/config/app/kumactl/v1alpha1"
 	util_http "github.com/kumahq/kuma/pkg/util/http"
+	kuma_version "github.com/kumahq/kuma/pkg/version"
 )
 
 func ApiServerClient(coordinates *config_proto.ControlPlaneCoordinates_ApiServer, timeout time.Duration) (util_http.Client, error) {
-	headers := make(map[string]string)
+	headers := map[string]string{
+		"User-Agent": fmt.Sprintf("Kumactl/%s (%s; %s) %s/%s",
+			kuma_version.Build.Version,
+			runtime.GOOS,
+			runtime.GOARCH,
+			kuma_version.Build.Product,
+			kuma_version.Build.GitCommit[:7]),
```

Comment:
I think to be compliant with user agent we need everything in the "comment section": https://bizcoder.com/the-much-maligned-user-agent-header/. (Even though as this article points out few folks respect it :) )

Can we go with `kumactl/<version> (<os>; <arch>; <product>/<commit>)`?

Main reason is the `kuma_version.Build.Product` may have spaces.

Reply:
OK ! It's done 👍🏻  !